packages/sdk/src/sdk/services/EntityManager/EntityManager.ts

```diff
@@ -139,18 +140,15 @@ export class EntityManager implements EntityManagerService {
         blockHash: jsonResponse.receipt.blockHash,
         blockNumber: jsonResponse.receipt.blockNumber
       }
-    } else if (response.status === 429) {
-      this.logger.error(
-        'API Rate Limit Exceeded: You have exceeded the allowed number of requests for this action. Please wait and try again later. If you require a higher rate limit, please send an email to api@audius.co with your request, detailing the reasons and expected usage.'
-      )
-      throw new Error(
-        'Error making relay request: API Rate Limit Exceeded. If you require a higher rate limit, please send an email to api@audius.co with your request.'
-      )
     } else {
-      throw new Error(
-        `Error making relay request ${response.status} ${
-          jsonResponse?.error?.message ? `: ${jsonResponse.error.message}` : '.'
-        }`
+      if (response.status === 429) {
```

Comment:
was this 429 case just not getting triggered before?

Reply:
I think the original code here would trigger it. But since this code is eating the original errors and just throwing a more generic message tailored to the status code, it doesn't work with the signup sagas (which have their own logic about what to do in response to the various status codes.
I left this conditional here because this is SDK and it's useful to see a rate limit error message if using SDK programmatically. But functionally, this isn't much different than it was before. Non-success status codes result in an Error being thrown. But now they will be `ResponseError`s, which include the status code and original message from the fetch.